webapp/channels/src/components/channel_header_dropdown/menu_items/close_channel/close_channel.test.tsx

```diff
@@ -16,30 +16,36 @@ describe('components/ChannelHeaderDropdown/MenuItem.CloseChannel', () => {
         },
     };
 
-    it('should match snapshot', () => {
-        const wrapper = shallow<CloseChannel>(<CloseChannel {...baseProps}/>);
+    it("should match snapshot", () => {
```

Comment:
We use single quotes instead of double quotes, so probably this will fail on the linter.

You can fix all linter issues by running `npm run fix` in the channels folder.